app/controllers/placements/organisations_controller.rb

```diff
@@ -30,10 +30,10 @@ def organisation
 
   def landing_page_path(organisation)
     if organisation.is_a?(School)
-      if organisation.expression_of_interest_completed?
-        placements_school_placements_path(organisation)
-      else
+      if organisation.current_hosting_interest(academic_year: current_user.selected_academic_year).blank? || !organisation.expression_of_interest_completed?
```

Comment:
for the sake of readability, might be nice to extract this condition (`organisation.current_hosting_interest(academic_year: current_user.selected_academic_year).blank? || !organisation.expression_of_interest_completed?`) to a method with a name expressing its intention.